docs/lineage/manual-lineage-feature-guide.md

```diff
@@ -0,0 +1,123 @@
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# About DataHub Manual Lineage
+
+<FeatureAvailability/>
+
+Starting in version `0.9.5`, DataHub supports the manual editing of lineage between entities in the UI. Add or remove upstream and downstream lineage edges in both the Lineage Visualization screen as well as the Lineage tab on entity pages. Use this feature to supplement automatic lineage extraction in ingestion or establish important entity relationships in sources that we don't support automatic extraction for yet! Manual lineage is only supported for Datasets, Charts, Dashboards, and Data Jobs for now.
```

Comment:
Maybe we can say "human editing of lineage" or "editing lineage by hand" ... I think we've gotten to a place where when we say "Manual" people get what we mean.